### PR TITLE
recipe data

### PR DESCRIPTION
**Pull request by sakshiarora04**

### CODE REVIEW

File: controllers/home-routes.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+//const withAuth = require('../utils/auth');
 
 router.get('/login', (req, res) => {
   // If the user is already logged in, redirect the request to another route
   if (req.session.logged_in) {
     res.redirect('/profile');
     return;
   }
-  res.render('login');
+  res.render('login',{login: true});
```
Comment (anUF0) [inline controllers/home-routes.js:11]: Please change this variable to 'loggedIn' or something similar
Makes the code less repetitive and more clear

File: controllers/api/recipeRoutes.js [hunk 2]
```diff
@@ -2,37 +2,20 @@ const router = require('express').Router();
...
     });
-
-    if (!recipeData) {
-      res.status(404).json({ message: 'No recipe found !' });
-      return;
-    }
-
-    res.status(200).json(recipeData);
+    const recipe = recipeData.get({ plain: true });
+    res.render('recipes', { recipe, logged_in: req.session.logged_in,login: true });
```
Comment (anUF0) [inline controllers/api/recipeRoutes.js:13]: See above

File: controllers/api/addRecipeRoutes.js [hunk 3]
```diff
@@ -38,6 +38,7 @@ router.get('/', async (req, res) => {
         user,
         categories,
         logged_in: req.session.logged_in,
+        login: true
```
Comment (anUF0) [inline controllers/api/addRecipeRoutes.js:41]: See below

File: views/layouts/main.handlebars [hunk 4]
```diff
@@ -85,6 +85,7 @@
           </ul>
         </div>
       </nav>
+     {{#unless login}}
```
Comment (anUF0) [inline views/layouts/main.handlebars:88]: see above

File: controllers/api/recipeRoutes.js [hunk 5]
```diff
@@ -2,37 +2,20 @@ const router = require('express').Router();
...
   } catch (err) {
     res.status(500).json(err);
   }
 });
 
-router.get('/', withAuth, async (req, res) => {
```
Comment (anUF0) [inline controllers/api/recipeRoutes.js:24]: Surely, we still need this route?
Or are we getting everything by PK?

**Review by Tristenh — APPROVED** [review]

**Pull request merged**